org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/quickfix/AbstractDeclarativeQuickfixProvider.java

```diff
@@ -37,7 +37,7 @@ protected Predicate<Method> getFixMethodPredicate(final String issueCode) {
 			@Override
 			public boolean apply(Method input) {
 				for (Fix annotation : input.getAnnotationsByType(Fix.class)) {
-					boolean result = annotation != null && issueCode.equals(annotation.value())
+					boolean result = annotation != null && annotation.value().equals(issueCode)
```

Comment:
Maybe use `Objects.equal(annotation.value(), issueCode)` to be really safe against NPEs?

Reply:
Ah good point. Done.